WordPress/metadata/release_notes.txt

```diff
@@ -1,8 +1,5 @@
-* [*] Block editor: Fixed an issue causing the editor to load indefinitely when sharing text from other apps or reblogging a post [https://github.com/wordpress-mobile/WordPress-Android/pull/15697]
-* [*] Added new About screen where the user can rate and share the app, visit our Twitter profile and blog, view other apps, and more. [https://github.com/wordpress-mobile/WordPress-Android/pull/15631]
-* [**] Block editor: Fix undo/redo functionality in links when applying text format. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4290]
-* [*] Block editor: Preformatted block: Fix an issue where the background color is not showing up for standard themes. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4292]
-* [**] Block editor: Update Gallery Block to default to the new format and auto-convert old galleries to the new format. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4315]
-* [***] Block editor: Highlight text: Enables color customization for specific text within a Paragraph block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4175]
-* [**] Reader: added a comment snippet below the post content; follow the conversation from there, then go to full comments to like and get involved. [https://github.com/wordpress-mobile/WordPress-Android/pull/15705]
-
+- Highlight text with any color in Paragraph blocks.
+- See and respond to comments after your post content in the Reader.
+- Create new Gallery blocks with the new design and auto-convert old galleries to match.
+- Undo and redo changes when formatting link text in blocks.
+- Load faster when reblogging posts and sharing text from other apps.
```

Comment:
When Google Translating this into French this leads to a translation which is 537 characters long.

Granted, this will in practice be translated by humans, which might come up with a shorter translation, but this still feels dangerously higher than the 500 limit in GooglePlay.

I've asked our freelancer in Slack if she could provide a shorter alternative version (that we'd then put in `release_notes_short.txt`). If she does it in time, that means we'll have a later PR adding the alternate shorter version to the `.po` file as well. If she does not do that in time, I'll simply remove the last item of the bullet point list for translated copies that exceed 500 chars at the time or the release, but it would be nicer to make this via the official way (`release_notes_short.txt`) beforehand if possible. Let's wait and see for her reply.